mysql/README.md

```diff
@@ -1,59 +1,131 @@
 # Overview
 
-Connect MySQL to Datadog in order to:
+The Datadog agent's MySQL check sends many database metrics to Datadog, including metrics for:
 
-* Visualize your database performance
-* Correlate the performance of MySQL with the rest of your applications
+* Query throughput
+* Query performance (average query run time, slow queries, etc)
+* Connections (currently open connections, aborted connections, errors, etc)
+* InnoDB (buffer pool metrics, etc)
 
-# Installation
+And [many more](https://github.com/DataDog/integrations-core/blob/master/mysql/metadata.csv). You can also invent your own metrics using custom queries.
 
-1. Create a ```datadog``` user with replication rights in your MySQL server:
+The MySQL check sends one service check: whether or not the Datadog agent can connect to MySQL.
 
-        sudo mysql -e "CREATE USER 'datadog'@'localhost' IDENTIFIED BY '<UNIQUEPASSWORD>';"
-        sudo mysql -e "GRANT REPLICATION CLIENT ON *.* TO 'datadog'@'localhost' WITH MAX_USER_CONNECTIONS 5;"
-2. Verify that the above commands worked by running:
+It does not send anything to your events stream.
 
+# Installation
 
-        mysql -u datadog --password=<UNIQUEPASSWORD> -e "show status" | \
-        grep Uptime && echo -e "\033[0;32mMySQL user - OK\033[0m" || \
-        echo -e "\033[0;31mCannot connect to MySQL\033[0m"
-        mysql -u datadog --password=<UNIQUEPASSWORD> -e "show slave status" && \
-        echo -e "\033[0;32mMySQL grant - OK\033[0m" || \
-        echo -e "\033[0;31mMissing REPLICATION CLIENT grant\033[0m"
+The MySQL check is included in the Datadog agent package, so simply install `datadog-agent` on your MySQL servers. If you need the newest version of the MySQL check, install the `dd-check-mysql` package; this package's check will override the one packaged with the agent. See the [integrations-core](https://github.com/DataDog/integrations-core#installing-the-integrations) repository for more details.
```

Comment:
Maybe link to the agent installation instructions: "so simply [install the Datadog agent](https://app.datadoghq.com/account/settings#agent) on your MySQL servers"